screens/ColorGeneratorScreen.js

```diff
@@ -1,45 +1,42 @@
 import React, { useState } from 'react';
-import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
 
 import { Button,  ColorBox } from '../components';
 
-const plusTen = (value) => {
-  return value <= 245 ? value + 10 : value
-};
-const minusTen = (value) => {
-  return value >= 10 ? value - 10 : value
-  };
 
+  const plusTen = (value) => {
+    return Math.min(255, value + 10)
+  };
+  const minusTen = (value) => {
+    return Math.max(0, value - 10)
+  };
+  const randomValue = () => {
+  return Math.floor(Math.random() * 256) + 1
+  };
 
-export default function ColorGeneratorScreen() {
 
-  const [RedValue, setRedValue] = useState(5);
-  const [GreenValue, setGreenValue] = useState(5);
-  const [BluValue, setBluValue] = useState(5);
-  const [GeneratedColor, setGeneratedColor] = useState("rgb(255, 255, 255)");
+export default function ColorGeneratorScreen({ navigation, props }) {
 
+  const [RedValue, setRedValue] = useState(randomValue);
+  const [GreenValue, setGreenValue] = useState(randomValue);
+  const [BluValue, setBluValue] = useState(randomValue);
 
-  const generatedBackgroundColor = () => {
-    setGeneratedColor(`rgb(${RedValue}, ${GreenValue}, ${BluValue} )`)
+  const rgbColor = {
+    backgroundColor: `rgb(${RedValue}, ${GreenValue}, ${BluValue} )`
   }
 
-  const randomValue = () => {
-
-    setRedValue(Math.floor(Math.random() * 256) + 1);
-
-    setGreenValue (Math.floor(Math.random() * 256) + 1);
-
-    setBluValue (Math.floor(Math.random() * 256) + 1);
-
+  const setRandomValue = () => {
+    setRedValue(randomValue);
+    setGreenValue(randomValue);
+    setBluValue(randomValue);
   }
 
     return (
-
-    <View style={[styles.container, { backgroundColor: GeneratedColor } ]}>
+      <View style={[styles.container, { backgroundColor: props }]}>
```

Comment:
Sorry. Somehow my answer to your question here didn't joined my review. 
Here it's decided that this View has some style, stored in `.container` AND there is a backgroundColor its value though is not yet specified, instead there are props used as a "place holder" which allowed me to declare backgroundColor's value later on .

Reply:
Well.. 

-  answer to your second question : Props are objects used to pass data between components, of which data flow is Uni-directional ( from Parent to child to grandchild ) and the only action you can do with the data you get from props is read it . How decent is this explanation ? 
- to answer your first question : it does absolutely nothing. There for it has been removed in my last commit. Here a summery of my process to get to this conclusion : 
after consol.logging what I thought is the  backgroundColor prop that I'm passing to the child component 
<img width="390" alt="Screenshot 2021-11-03 at 16 24 42" src="https://user-images.githubusercontent.com/43641321/140128151-fcb4ea4f-ed64-4a3a-a2db-7486fc313834.png">
I looked at this out put getting this confirmation: I cannot be passing anything to the child because in my 1st log backgroundColor has an undefined value but then in my 2nd log inside the child backgroundColor has the correct value that I wanted , how is that possible ? well because I'm using params to pass data to the child there fore props are unnecessary here.